WolvenKit.App/Models/ProjectManagement/CP77Mod.cs

```diff
@@ -26,7 +26,9 @@ public CP77Mod(Cp77Project project)
             Version = project.Version;
 
             OpenProjectFiles = project.OpenProjectFiles.OrderBy(x => x.Key)
-                .Select(x => x.Value).ToList();
+                .Select(x => x.Value)
```

Comment:
This doesn't matter. it should be changed in SaveOpenFilePaths() in ProjectExplorerViewModel.cs